ansible/roles/shared_dir/tasks/install.yml

```diff
@@ -1,5 +1,10 @@
 ---
 
+- name: Update the apt cache
+  when: shared_drive_enabled
+  apt:
+    update_cache: yes
+
```

Comment:
Since it uses with_items would that try to call `apt-get update` twice? that was the original reason I moved it here